vars/provisionNodes.groovy

```diff
@@ -163,236 +126,177 @@ def call(Map config = [:]) {
                       if [ -d /tmp/Functional_*/ ]; then
                           rm -rf /tmp/Functional_*
                       fi'''
+  new_config['pre_reboot'] = cleanup_logs
+  new_config['post_reboot'] = cleanup_logs
```

Comment:
Do we actually need these if we are provisioning nodes fresh every time?

Reply:
But ultimately, any post-test state will be wiped by the re-provision so i'm not sure i see the value here.